src/cli/exec.rs

```diff
@@ -56,6 +55,9 @@ pub struct Args {
 
     #[clap(flatten)]
     pub config: ConfigCli,
+
+    #[clap(flatten)]
+    pub solver_config: SolverConfig,
```

Comment:
Why is it not part of the lockfile update config?

Reply:
Because there are a number of CLIs that dont have a lockfileupdate config but do solve, like exec or adding a channel.